src/components/resources/misc/list/Namespaces.tsx

```diff
@@ -45,21 +55,27 @@ const Namespaces: React.FunctionComponent = () => {
     }
   }, [showPopover, refetch]);
 
-  return (
-    <React.Fragment>
-      <IonPopover isOpen={showPopover} event={popoverEvent} onDidDismiss={() => setShowPopover(false)}>
-        {isError ? (
-          <IonList>
-            <IonItem>{error ? error.message : 'Could not get Namespaces'}</IonItem>
-          </IonList>
-        ) : data ? (
-          <IonList>
-            <IonItem button={true} detail={false} onClick={() => setAllNamespaces()}>
-              {cluster && cluster.namespace === '' ? <IonIcon slot="end" color="primary" icon={checkmark} /> : null}
-              <IonLabel>All Namespaces</IonLabel>
-            </IonItem>
+  const [filterText, setFilterText] = useState('');
+  const filterRegex = new RegExp(filterText, 'gi');
 
-            {data.items.map((namespace, index) => {
+  function renderItems(items: V1Namespace[]) {
```

Comment:
Can we use arrow functions for the `renderItems`, `renderError` and `renderLoading` functions?

Reply:
Done.